python/lsst/ts/dimm/config_schema.py

```diff
@@ -26,7 +26,7 @@
 CONFIG_SCHEMA = yaml.safe_load(
     """
     $schema: http://json-schema.org/draft-07/schema#
-    $id: https://github.com/lsst-ts/ts_dimm/blob/master/python/lsst/ts/dimm/config_schema.yaml
+    $id: https://github.com/lsst-ts/ts_dimm/blob/master/python/lsst/ts/dimm/config_schema.py
```

Comment:
Very minor suggestion, but I think the history would be a bit easier to follow if you squashed this with the prior commits, so that comfig_schema.py is added in one commit in its final form.